docs/administration/how_to_guides/organization_management/set_up_scim.mdx

```diff
@@ -0,0 +1,210 @@
+
+System for Cross-domain Identity Management (SCIM) is an open standard that allows for the automation of user provisioning. Using SCIM, you can automatically provision and deprovision users in your LangSmith organization and workspaces, keeping user access synchronized with your organization's identity provider.
+
+:::note
+SCIM is available for organizations on the [Enterprise plan](https://www.langchain.com/pricing). Please [contact sales](https://www.langchain.com/contact-sales) to learn more.
+
+While in Beta, SCIM support is API-only (see instructions below).
+:::
+
+## What is SCIM?
+
+SCIM enables automatic user provisioning and deprovisioning between your identity provider (IdP) and LangSmith. This eliminates the need for manual user management and ensures that user access is always up-to-date with your organization's identity system.
+
+## Benefits of SCIM
+
+- **Automated user management**: Users are automatically added, updated, and removed from LangSmith based on their status in your IdP
+- **Reduced administrative overhead**: No need to manually manage user access across multiple systems
+- **Improved security**: Users who leave your organization are automatically deprovisioned from LangSmith
+- **Consistent access control**: User attributes and group memberships are synchronized between systems
+- **Scalable**: Efficiently manage large teams with many workspaces and custom roles
+
+## Prerequisites
+
+- Your organization must be on an Enterprise plan
+- Your Identity Provider (IdP) must support SCIM 2.0
+- Only [Organization Admins](../../concepts#organization-roles) can configure SCIM
+- For cloud customers: [SAML SSO](./set_up_saml_sso.mdx) must be configured for your organization
+- For self-hosted customers: [OAuth with Client Secret](../../../self_hosting/configuration/sso#with-secret) authentication mode must be enabled
+
+## Capabilities
+
+SCIM enables the following capabilities:
+
+- **User provisioning**: Automatically add users to your LangSmith organization
+- **User deprovisioning**: Automatically remove users from your organization
+- **Attribute synchronization**: Keep user attributes (like full name) synchronized between your IdP and LangSmith
+- **Group-based access**: Sync membership from IdP user groups to LangSmith workspaces
+- **Role assignment**: Select specific [Organization Roles](../../concepts#organization-roles) and [Workspace Roles](../../concepts#workspace-roles) for groups of users
+
+## Group Naming Convention
+
+Group membership maps to LangSmith Workspace membership and workspace roles with a specific naming convention:
+
+### Organization Admin Groups
+
+- `LS:Organization Admin` or `LS:Organization Admins`
+
+### Workspace-Specific Groups
+
+Format: `LS:<org_role_name>:<workspace_name>:<workspace_role_name>`
+
+Examples:
+
+- `LS:Organization User:Production:Annotators`
+- `LS:Organization User:Engineering:Developers`
+- `LS:Organization User:Marketing:Viewers`
+
+### Role Precedence
```

Comment:
This part is pretty important (and easy to miss). Maybe bring this up to a top level section above "Group Naming Convention"